### ISSUE TITLE
Error building puzzle; no exit door

### ISSUE THREAD

**Issue by Blacksilver42** [BEEmod/BEE2.4#677]
## *Grumble grumble*

Note: I am writing this post as it happens, not after the fact.

After having the same problem as #351, I did the workaround suggested by @TeamSpen210:

1. go to `Portal 2/bin/`
2. Delete `vbsp.exe` and `vrad.exe`
3. Go to steam and verify game files
4. re-export in BEE2

## And an empty test chamber worked. Yay!

I added a *single trigger once* to an existing test chamber. **Build error.** %#&$.
I deleted the Trigger once and rebuilt. **Build error.** $&@#.

So I delete `vbsp.exe` and `vrad.exe`, verify game files (2 files failed to validate), and re-export in BEE2.4.

Portal 2 happily starts, and I spool up the editor.
I open a previous test chamber, which was totally stock, and build it. **Build error**.

I open a brand new, out-of-the-box chamber. It builds!
Aw maaaaaaaan. More problems.

## The door is... weird.
![](http://i.imgur.com/uAhbZeM.jpg)
[I took more screenshots](http://imgur.com/a/HaCq5)

So here's the entire console up to this point: http://pastebin.com/8HTbgGd7

I add some **non-BEE** items to a chamber.

http://imgur.com/a/X4xhj

Build error. dammit.

I delete all the shebang. Happily builds.

So I go through, deleting things one by one, MCVE-ing, trying to find the source of the problem. [Screenshots](http://imgur.com/a/Xcmjo)

[The entrance door is being weird now too.](http://imgur.com/a/hFIH7)

## It's the large floor button.
![image](https://user-images.githubusercontent.com/14184826/27740148-f10ad5aa-5d7e-11e7-92d4-9ec1de69dc91.png)
This builds.

anyway, I gtg now. kthxbai

**Comment by Blacksilver42** [comment]
I *may* have found the problem.

The first time I ran BEE2, I had mistakenly put the zipped packages in the main folder.

**Comment by Blacksilver42** [comment]
Norton.

**Comment by Nikita74RU** [comment]
Hey

**Comment by Blacksilver42** [comment]
tl;dr check your antivirus.
